docs/development/doc_guide.rst

```diff
@@ -342,24 +342,30 @@ Sphinx extensions
 `PlasmaPy's documentation`_ is built with the following Sphinx_
 extensions:
 
-* `sphinx.ext.autodoc` for including documentation from docstrings
-* `sphinx.ext.intersphinx` for linking to other projects' documentation
-* `sphinx.ext.graphviz` to allow Graphviz_ graphs to be included
-* `sphinx.ext.mathjax` for math rendering with MathJax_
-* `sphinx.ext.napoleon` for allowing NumPy style docstrings
-* `sphinx.ext.todo` to support ``todo`` :term:`directive`\ s
-* nbsphinx_ for including Jupyter_ notebooks
-* `sphinx_copybutton`_ to add a "copy" button for code blocks
-* `sphinx_gallery.load_style`_ for using sphinx-gallery styles
-* `IPython.sphinxext.ipython_console_highlighting`_
-* `sphinx_changelog`_ for rendering `towncrier`_ changelogs
+* `sphinx.ext.autodoc` for including documentation from docstrings.
+* `sphinx.ext.intersphinx` for linking to other projects' documentation.
+* `sphinx.ext.graphviz` to allow Graphviz_ graphs to be included.
+* `sphinx.ext.mathjax` for math rendering with MathJax_.
+* `sphinx.ext.napoleon` for allowing NumPy style docstrings.
+* `sphinx.ext.todo` to support ``todo`` |directives|.
+* |nbsphinx|_ for including Jupyter_ notebooks.
+* |sphinxcontrib-bibtex|_ to enable usage of a BibTeX_ file to create
+  the :doc:`../bibliography`.
+* |sphinx_copybutton|_ to add a "copy" button for code blocks.
+* |sphinx_gallery.load_style|_ for using sphinx-gallery styles.
+* |IPython.sphinxext.ipython_console_highlighting|_.
+* |sphinx_changelog|_ for rendering `towncrier`_ changelogs.
 * `plasmapy_sphinx` for customizations created for use in PlasmaPy and
   affiliated packages. Note that `plasmapy_sphinx` is expected to be
   broken out into its own package in the future.
 
 These extensions are specified in :confval:`extensions` configuration
 value in |docs/conf.py|_.
 
+When an extension contains new |roles| or |directives|, it may be
+necessary to add them to ``rst-roles`` and ``rst-directives`` in the
+``[flake8]`` section of |setup.cfg|_ to avoid linter errors.
```

Comment:
Give the reason why.  If I remember correctly its so the linter check does not fail, but I don't remember which one.

Reply:
- [ ] Put a little more detail on this being for CI in PRs